src/StockportWebapp/Views/stockportgov/Directory/Results.cshtml

```diff
@@ -19,7 +19,7 @@
         <div class="grid-container">
             <div class="grid-100 directory-sub-title">
                 <h1>Results for @Model.Directory.Title</h1>
-                <span class="directory-result-count">Showing @Model.FilteredEntries.Count() results in @Model.Directory.Title</span>
+                <span class="directory-result-count">Showing <strong>@Model.FilteredEntries.Count()</strong> results in <strong>@Model.Directory.Title</strong></span>
```

Comment:
remove <strong> tag and replace with <b> or use spans with classes and font-weight